helpers/README.md

```diff
@@ -0,0 +1,353 @@
+
+# Z64 Helpers
+
+By default these aren't set `static inline`. To do this, you can `#define` `_Z64HDR_HELPER_PREFIX_` to achieve this.
+```c
+#define _Z64HDR_HELPER_PREFIX_ static inline
+#include <helpers/Helper.h>
+```
+---
+### List of content
+1. [MATRIX](#matrix)
+	1. [Matrix Multiply](#matrix-mult)
+	1. [Matrix Macros](#matrix-macros)
+1. [PHYSICS](#physics)
+	1. [PhysicsStrand Struct](#physicsstrand-struct)
+		1. [PhysicsInfo](#physicsinfo)
+		1. [PhysicsHead](#physicshead)
+		1. [PhysicsGfx](#physicsgfx)
+		1. [PhysicsSpheres](#physicsspheres)
+		1. [PhysicsConstraint](#physicsconstraint)
+		1. [PhysicsRigidity](#physicsrigidity)
+    1. [Examples](#examples)
+        1. [PhysicsStrand](#physicsstrand)
+        1. [ActorInit](#actorinit)
+        1. [OverridePostDraw](#overridepostdraw)
+		1. [PhysicCallback](#physiccallback)
+        1. [ActorDraw](#actordraw)
+1. [Credits](#credits)
+---
+# Matrix
+## Matrix Mult
+```c
+void Matrix_MultX(f32 scale, Vec3f* dst);
+void Matrix_MultY(f32 scale, Vec3f* dst);
+void Matrix_MultZ(f32 scale, Vec3f* dst);
+```
+[Back to top](#list-of-content)
+
+---
+## Matrix Macros
+```c
+#define Matrix_RotateY_s(binang, A) Matrix_RotateY(BINANG_TO_RAD(binang), A)
+#define Matrix_RotateX_s(binang, A) Matrix_RotateX(BINANG_TO_RAD(binang), A)
+#define Matrix_RotateZ_s(binang, A) Matrix_RotateZ(BINANG_TO_RAD(binang), A)
+
+#define Matrix_RotateY_f(degf, A)   Matrix_RotateY(DEGF_TO_RADF(degf), A)
+#define Matrix_RotateX_f(degf, A)   Matrix_RotateX(DEGF_TO_RADF(degf), A)
+#define Matrix_RotateZ_f(degf, A)   Matrix_RotateZ(DEGF_TO_RADF(degf), A)
+```
+[Back to top](#list-of-content)
+
+---
+# Physics
+---
+## PhysicsStrand Struct
+```md
+typedef struct {
+	PhysicsInfo       info;
+	PhysicsHead       head;
+	PhysicsGfx        gfx;
+	PhysicsSpheres    spheres; // "collision" spheres
+	PhysicsConstraint constraint;
+	PhysicsRigidity   rigidity;
+	f32 limbsLength[];
+} PhysicsStrandInit;
```

Comment:
would be nice to add information, but if it's just comments may as well leave them in the source rather than having to maintain duplicates of the structure definition (same for all structs in the readme)